augly/utils/expected_output/image_tests/expected_metadata.json

```
@@ -24,6 +24,36 @@
             "src_width": 1920
         }
     ],
+    "distort_barrel": [
+        {
+            "dst_height": 1080,
+            "dst_width": 1920,
+            "intensity": 100.0,
+            "name": "distort_barrel",
+            "output_path": null,
+            "a": 0.1,
+            "b": 0.0,
+            "c": 0.0,
+            "d": 1.0,
+            "src_height": 1080,
+            "src_width": 1920
```

```suggestion
            "a": 0.1,
            "b": 0.0,
            "c": 0.0,
            "d": 1.0,
            "dst_height": 1080,
            "dst_width": 1920,
            "intensity": 100.0,
            "name": "distort_barrel",
            "output_path": null,
            "src_height": 1080,
            "src_width": 1920
```
nit: keep these alphabetized like the other entries.